cmd/influxdb-writer/main.go

```diff
@@ -16,8 +16,8 @@ import (
 	consumerTracing "github.com/mainflux/mainflux/consumers/tracing"
 	"github.com/mainflux/mainflux/consumers/writers/api"
 	"github.com/mainflux/mainflux/consumers/writers/influxdb"
-	influxDBClient "github.com/mainflux/mainflux/internal/clients/influxdb"
-	jaegerClient "github.com/mainflux/mainflux/internal/clients/jaeger"
+	influxdbclient "github.com/mainflux/mainflux/internal/clients/influxdb"
```

Comment:
maintain consistency in the naming. In influxdb-reader it is` influxDBclient`

Reply:
Resolved